### PR TITLE
 Standard

### LINKED ISSUE #37: standard

### PR DESCRIPTION
**Pull request by extremeheat**
Fix #37

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+examples/old
```
Comment (rom1504) [inline .eslintignore]: Why ?

Comment (extremeheat) [reply .eslintignore]: removed the old examples

File: .eslintignore [hunk 2]
```diff
@@ -0,0 +1 @@
+test/
```
Comment (rom1504) [inline .eslintignore:1]: why no linting for tests ?

**Review by rom1504 — APPROVED** [review]

**Comment by rom1504** [comment]
ok hopefully we don't need such big PRs now

File: .eslintignore (hunk 2 above)
Comment (extremeheat) [reply .eslintignore:1]: had some generated code there, I'll move things around in an upcoming PR

Comment (rom1504) [reply .eslintignore:1]: thanks

**Pull request merged**